src/renderers/WebGLRenderer.js

```diff
@@ -191,11 +192,17 @@ function WebGLRenderer( parameters ) {
 		_canvas.addEventListener( 'webglcontextlost', onContextLost, false );
 		_canvas.addEventListener( 'webglcontextrestored', onContextRestore, false );
 
-		_gl = _context || _canvas.getContext( 'webgl', contextAttributes ) || _canvas.getContext( 'experimental-webgl', contextAttributes );
+		var webglVersion = ( _webglVersion === 'webgl2' || _webglVersion === 'auto' ) ? 'webgl2' : 'webgl';
```

Comment:
Instead of adding a `webglVersion` parameter to `WebGLRenderer` constructor...
Can you modify the `webgl2_sandbox` so it looks like this:

```js
canvas = document.createElement( 'canvas' );
context = canvas.getContext( 'webgl2' );
renderer = new THREE.WebGLRenderer( { canvas: canvas, context: context } );
```

Reply:
> The less APIs we introduce, the less APIs we have to maintain forever

Agreed. And we'll switch to `WebGL2Context` as the default once WebGL2.0 on Three.js is stable, won't we?